doc/pages/upgrade/030-upgrade_100_101.md

```diff
@@ -0,0 +1,211 @@
+TLS termination, while the latter set the secret containing the TLS certificate (further details on
+certificates can be found [here](062-handling_certificates.html)).
+
+To summarize, the needed changes will look like the following sample snippet:
+```yaml
+apiVersion: api.astarte-platform.org/v1alpha2
+kind: Astarte
+...
+spec:
+  ...
+  vernemq:
+    sslListener: true
+    sslListenerCertSecretName: <the-tls-certificate-name>
+    ...
+  version: 1.0.1
+```
+
+Once the yaml file is applied, the Operator will take over ensuring the reconciliation of your
+Astarte instance.
+
+### Caveats for Astarte Flow
+
+Currently, although [Astarte Flow](https://docs.astarte-platform.org/flow/) is a component of
+Astarte, it doesn't follow Astarte's release cycle. Therefore if you upgraded your Astarte instance
+to v1.0.0, Astarte Operator will try to deploy `astarte/astarte_flow:1.0.0` which is currently not
+existent.
+
+All you have to do to overcome this temporary limitation is to edit your Astarte resource by
+explicitly setting the Astarte Flow image you plan to use:
+```yaml
+spec:
+  ...
+  components:
+    ...
+    flow:
+      image: <the-astarte-flow-image>
+```
+
+All the available Astarte Flow's tags can be found
+[here](https://hub.docker.com/r/astarte/astarte_flow/tags?page=1&ordering=last_updated).
+
+## Deploy `AstarteDefaultIngress` in place of `AstarteVoyagerIngress`
+
+The current section describes the procedure for replacing the deprecated `AstarteVoyagerIngress`
+with the new `AstarteDefaultIngress`. If the Voyager ingress is not deployed within your cluster,
+feel free to skip this section.
+
+The first step requires the installation of the [NGINX Ingress
+Controller](https://kubernetes.github.io/ingress-nginx/):
+
+```bash
+$ helm install ingress-nginx ingress-nginx/ingress-nginx -n ingress-nginx \
+    --set controller.service.externalTrafficPolicy=Local \
+    --create-namespace
+```
+
+Then the `AstarteDefaultIngress` must be deployed. If your Kubernetes installation supports
+LoadBalancer ingresses (most managed ones do), you should be able to get away with the most standard
+CR:
+
+```yaml
+apiVersion: ingress.astarte-platform.org/v1alpha1
+kind: AstarteDefaultIngress
+metadata:
+  name: adi-example-minimal
+  namespace: astarte
+spec:
+  ### Astarte Default Ingress CRD
+  astarte: astarte
+  tlsSecret: <your-astarte-tls-cert>
+  api:
+    exposeHousekeeping: true
+  dashboard:
+    deploy: true
+    ssl: true
+    host: <your-astarte-dashboard-host>
+  broker:
+    deploy: true
+```
+
+Further details on this topic can be found at [this page](064-setup_astartedefaultingress.html).
+
+When the `AstarteDefaultIngress` resource is created, the Operator is responsible for creating an
+NGINX ingress and a service of type LoadBalancer in the Astarte namespace.
+
+Wait until both the ingress and the new service have an assigned external IP, then update your Cloud
+DNS zone record sets with the new external IPs.
```

Comment:
IMO after this sentence we need to specify:
1. How to retrieve the external IPs and which IP to use for each DNS record
2. After setting the DNS records you have to be sure that the records are propagated, for devices as well. One way to do this is to wait for the records TTL before deleting the AVI.